packages/web3-providers/src/SimpleHash/apis/EVM.ts

```diff
@@ -416,7 +416,7 @@ export class SimpleHashAPI_EVM implements NonFungibleTokenAPI.Provider<ChainId,
                     hash: x.transaction,
                     from: x.from_address,
                     token_id: x.token_id,
-                    transaction_link: new ExplorerResolverAPI().transactionLink(chainId, x.transaction),
+                    transaction_link: new ExplorerResolverAPI().transactionLink(chainId, x.transaction)!,
```

Comment:
The explorer link could be empty when the user add a custom network. Since we force it not null is not safe.